docs/xdr/features/collect/ingestion_methods/sekoiaio_docker_concentrator.md

```diff
@@ -0,0 +1,435 @@
+
+2. Verify if this information is taken into account by the concentrator. At start-up, the concentrator always shows the list of Intakes with the protocols and ports.
+    ```bash
+    sudo docker compose logs | more
+    ```
+
+3. Check that you correctly declared the `ports` section in the `docker-compose.yml` file. They MUST be the same as the ports declared in the `intakes.yaml` file. For instance, if you declared 4 technologies on ports `25020`, `25021`, `25022` and `25023`, the ports line the `docker-compose.yml` has to be `"25020-25023:25020-25023"`. 
+
+4. Verify that traffic is incoming from your log source, meaning no firewall is blocking the events.
+    ```bash
+    sudo tcpdump -i <change_with_interface_name> -c10 -nn src <remote_ip> -vv
+    ```
+
+    To find those values:
+
+    - `change_with_interface_name`use the command `ip addr`
+    - `remote_ip`is the IP from which the logs should be incoming 
+
+### Step 2: verify everything is correctly configured to forward events to SEKOIA.IO
+
+1. Check the Intake keys you wrote in `intakes.yaml` are correct.
+
+2. Check the network flow between the concentrator host and SEKOIA.IO is opened to the destination `intake.sekoia.io` on protocol `TCP` and port `10514`. You can easily check it with `telnet`:
+    ```bash
+    sudo apt install telnet && telnet intake.sekoia.io 10514
+    ```
+
+    This command should return the following answer, meaning the network flow is opened:
+
+    ```bash
+    Connected to intake.sekoia.io.
+    Escape character is '^]'.
+    ```
+
+    Remove `telnet` if don't need it anymore
+
+    ```bash
+    sudo apt remove telnet
+    ```
+
+3. Finally check the status of the SEKOIA.IO plateform on [https://status.sekoia.io](https://status.sekoia.io).
+
+## Additional information
+
+The image used to run the concentrator is maintained on [this github repository](https://github.com/SEKOIA-IO/sekoiaio-docker-concentrator). Feel free to contribute and make a pull request to improve the concentrator!
+
+## How to update the concentrator
+Docker uses the notion of tag to identify the version of an image. The tag is always referenced in line starting with `image` in `docker-compose.yml`:
+
+```
+image: ghcr.io/sekoia-io/sekoiaio-docker-concentrator:0.9
+```
+
+`0.9` means the version used by `docker compose` is 0.9. You can find all the versions available on the GitHub repository [here](https://github.com/SEKOIA-IO/sekoiaio-docker-concentrator/pkgs/container/sekoiaio-docker-concentrator/versions?filters%5Bversion_type%5D=tagged)
+
+To update the concentrator, just change the tag in `docker-compose.yml`, then recreate the concentrator with the command:
+```bash
+sudo docker compose up -d
+```
+
+There is a special tag called `latest` which always refers to the latest version. If you use this tag, every time the concentrator will be recreated, the image will be updated with the latest version.

```

Comment:
```suggestion
There is a special tag called `latest` which always refers to the latest version. If you use this tag, every time the concentrator will be re-created and the image will be updated to the latest version.

```

Reply:
I don't agree with you on this one. I mean "Every time the concentrator is recreated => This image is updated"